image-viewer.py

```diff
@@ -55,5 +56,72 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
+        
+    def create_menu(self):
+        menubar = Menu(self.root)
+        self.root.config(menu=menubar)
+        
+        file_menu = Menu(menubar, tearoff=False)
+        file_menu.add_command(label='Open Folder', command=self.open_folder, underline=0)
+        file_menu.add_command(label='Refresh', command=self.refresh, underline=0)
+        file_menu.add_command(label='Exit', command=self.root.destroy, underline=1)
+        menubar.add_cascade(label='File', menu=file_menu, underline=0)
+    
+    def open_folder(self):
+        new_dir = filedialog.askdirectory()
+        
+        if new_dir != "": 
+            self.img_dir = new_dir
+            self.find_images(self.img_dir)
+    
+    def refresh(self):
+        self.find_images(self.img_dir)
+    
+
+    def find_images(self, dir):
+        
+        self.image_files = []
+        self.current_image_index = 0
+        
+        for item in listdir(dir):
+            if isfile(f"{dir}/{item}"):
+                if(item.lower().endswith(('.png', '.jpg', '.jpeg', '.ico', '.gif'))):
+                    self.image_files.append(item)
+                
+        self.display_image()
+
+
+    def display_image(self):
+        
+        try:
+            self.name_label.configure(text=self.image_files[self.current_image_index])
+            new_img = Image.open(f"{self.img_dir}/{self.image_files[self.current_image_index]}")
+            self.current_image = ImageTk.PhotoImage(self.resize_image(new_img))
+            self.display.configure(image=self.current_image)
+            
+        except OSError as e:
+            self.name_label.configure(text=e)
+            self.current_image = ImageTk.PhotoImage(self.blank_img)
+            self.display.configure(image=self.current_image)
+            
+        except IndexError:
+            self.name_label.configure(text="No image files found")
+            self.current_image = ImageTk.PhotoImage(self.blank_img)
+            self.display.configure(image=self.current_image)
+    
+    # Resize the images for display if they are bigger than 500 x 300 -- keeping their aspect ratio
```

Comment:
I debate the value of this comment - I feel like `Resize the images for display if they are bigger than 500 x 300` is represented in the code, but `keeping their aspect ratio` helps explain the _why_, which provides value.

Maybe a `# keeping their aspect ratio` comment could be above the if statements on line 114 instead?